plugins/frontend-search/README.md

```diff
@@ -13,54 +12,60 @@ plugins=( ... frontend-search)
 
 ```
 
-
-## Usage ##
+## Usage
 
 You can use the frontend-search plugin in these two forms:
 
-* `frontend <context> <term> [more terms if you want]`
-* `<context> <term> [more terms if you want]`
+- `frontend <context> <term> [more terms if you want]`
+- `<context> <term> [more terms if you want]`
 
 For example, these two are equivalent:
 
 ```zsh
-$ frontend angularjs dependency injection
-$ angularjs dependency injection
+$ frontend angular dependency injection
+# Will turn into ...
```

Comment:
Reverting that back. Thanks, @mcornella

Reply:
I meant that `angular ...` turns into `frontend angular ...`, not the other way round: you can reverse the order or delete the `will turn into` line. Using `angular` instead of `angularjs` is ok though.